src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-color-picker {
+  width: 250px;
+  border-radius: 6px;
+  box-shadow: $iui-elevation-2;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-top: 10px;
+  @include themed {  background: rgb(255, 255, 255);}
+
+
+  .iui-color-option {
+    height: 20px;
+    width: 20px;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    border-radius: 4px;
+
+    &.iui-light {
+      box-shadow: $iui-elevation-2;
+    }
+
+    &.iui-active {
+      @include themed {
+        box-shadow: 0 0 0 2pt rgb(100, 149, 237);
+      }
+    }
+
+    &:hover { 
+      @include themed {
+        box-shadow: 0 0 0 2pt rgb(170, 170, 170);
+      }
+    }
+
+    &:focus,
+    &:focus-visible {
```

Comment:
This will break in Safari (and IE). And even in other browsers, it will show focus effect on click (which is wrong).

Recommend reading this: [https://www.tpgi.com/focus-visible-and-backwards-compatibility](https://www.tpgi.com/focus-visible-and-backwards-compatibility/#:~:text=the%20most%20viable%20(though%20still%20not%20particularly%20elegant)%20solution%20may%20be%20to%20use%20the%20%3Anot()%20negation%20pseudo-class%2C%20and%20to%20(paradoxically)%20define%20styles%20not%20for%20%3Afocus-visible%2C%20but%20to%20undo%20%3Afocus%20styles%20when%20it%20is%20absent.)